setup.py

```diff
@@ -25,7 +25,7 @@
             "FullyConnected = modulus.models.mlp:FullyConnected",
             "Pix2Pix = modulus.models.pix2pix:Pix2Pix",
             "One2ManyRNN = modulus.models.rnn:One2ManyRNN",
-            #'SphericalFourierNeuralOperatorNet = modulus.models.sfno:SphericalFourierNeuralOperatorNet',
+            "SphericalFourierNeuralOperatorNet = modulus.experimental.models.sfno:SFNO",
```

Comment:
Why not just `SFNO` like the other models above?

Reply:
Fixed it